src/layout/RepeatingGroup/config.ts

```diff
@@ -190,6 +204,17 @@ export const Config = new CG.component({
         .optional(),
     ),
   )
+  .addProperty(
+    new CG.prop(
+      'pagination',
+      new CG.obj(new CG.prop('rowsPerPage', new CG.int().setMin(1)))
+        .optional()
+        .setTitle('Pagination options')
+        .setDescription(
+          'Pagination options for the repeating group table. Not applicable when `edit.mode` is `showAll`',
```

Comment:
Looks like it now works even with `showAll`? Maybe this comment was written before then. :innocent:

Reply:
Ahh yes, woops 🙈